ChangeLog.d/ssl_cnsa_preset.txt

```diff
@@ -0,0 +1,2 @@
+Features
+   * Add support for CNSA preset that supersedes NSA SuiteB.
```

Comment:
The CI complains that the file is missing a newline at the end. Please add it. It's ok to amend the commit.

Reply:
Yes, fixed that locally, was waiting for CI to complete and see if there are any other issues, `make test` failed some dependencies. Thanks, will amend .

Reply:
pushed with amended fix for changelog and also a depends.py failure.